airbyte-integrations/connectors/source-snowflake/src/main/resources/spec.json

```diff
@@ -52,11 +52,46 @@
               }
             }
           },
+          {
+            "title": "Key Pair Authentication",
+            "type": "object",
+            "order": 1,
+            "required": ["username", "private_key"],
+            "properties": {
+              "auth_type": {
+                "type": "string",
+                "const": "Key Pair Authentication",
+                "order": 0
+              },
+              "username": {
+                "description": "The username you created to allow Airbyte to access the database.",
+                "examples": ["AIRBYTE_USER"],
+                "type": "string",
+                "title": "Username",
+                "order": 1
+              },
+              "private_key": {
+                "type": "string",
+                "title": "Private Key",
+                "description": "RSA Private key to use for Snowflake connection. See the <a href=\"https://docs.airbyte.com/integrations/destinations/snowflake\">docs</a> for more information on how to obtain this key.",
```

Comment:
To be more accurate, please use this link

https://docs.airbyte.com/integrations/destinations/snowflake#key-pair-authentication

@evantahler I think we need to copy that particular key pair setup section of the doc into the Snowflake source documentation page as well, and then link it to the source page instead. @aharbunou-branch would you like to help update the doc in the PR too?

Reply:
@theyueli I've updated documentation for source snowflake connector. I also updated changelog and bumped connector version. Let me know if it looks good.

Reply:
Thank you! looks good!